src/lib/draw/AttributeMode.svelte

```diff
@@ -70,7 +70,9 @@
 
       // Extent of points is defined in a weird way, special-case it
       if (feature.geometry.type == "Point") {
-        $map.flyTo({ center: feature.geometry.coordinates });
+        $map.flyTo({
+          center: feature.geometry.coordinates as [number, number],
```

Comment:
See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/045220a0682a1f437c08610db4ad6eaf76b2f9f0/types/geojson/index.d.ts#L38
geojson types use `number[]` even though it should really be 2D